src/qiskit_qec/utils/decoding_graph_attributes.py

```diff
@@ -88,6 +111,26 @@ class DecodingGraphEdge:
     # TODO: Should code/decoder specific properties be accounted for when comparing edges
```

Comment:
Has this TODO been handled? If not please make appropriate issues/exceptions.

Reply:
it's not something from this PR, but it needn't be there so I removed it anyway